### PR TITLE
ENH: use fabio to perform fit2d mask saving and loading, with boolian…

### PR DESCRIPTION
**Pull request by CJ-Wright**
… switching to make certain masks end up in skbeam convention

### CODE REVIEW

**Comment by CJ-Wright** [comment]
@tacaswell @danielballan ready for review

**Comment by tacaswell** [comment]
Please put shims to not break API.

**Comment by tacaswell** [comment]
Is the existing writer broken?

**Comment by CJ-Wright** [comment]
The existing writer will only work with 2048x2048 images. What do you mean shims?

File: skbeam/io/fit2d_save.py [hunk 1]
```diff
@@ -1,120 +1 @@
-from __future__ import print_function, division
```
Comment (tacaswell) [inline skbeam/io/fit2d_save.py]: can you add a deprecation warning here directing people to the new location?

Comment (CJ-Wright) [inline skbeam/io/fit2d_save.py]: What is the syntax for that?

**Comment by ericdill** [comment]
Consider renaming `skbeam/io/fit2d_io.py` to `skbeam/io/fit2d.py`.  We don't need `io` in the import twice

**Comment by ericdill** [comment]
Otherwise :+1: for simplified code. Any idea how good the fabio conda package is working?

File: skbeam/io/fit2d_save.py (hunk 1 above)
Comment (CJ-Wright) [inline skbeam/io/fit2d_save.py]: I think that I have added the warning correctly.

**Comment by CJ-Wright** [comment]
Thanks @tacaswell !

**Pull request merged**